Cargo.toml

```diff
@@ -16,18 +16,22 @@ features = ["stm32g081", "rt"]
 default-target = "thumbv6m-none-eabi"
 
 [dependencies]
-cortex-m = "0.6.1"
+cortex-m = "0.6.3"
 nb = "0.1.1"
-stm32g0 = "0.10.0"
-as-slice = "0.1.2"
+as-slice = "0.1.3"
+
+[dependencies.stm32g0]
+path = "../stm32-rs/stm32g0"
+# version = "0.12.0"
+features = ["rt"]
 
 [dependencies.bare-metal]
-features = ["const-fn"]
-version = "0.2.5"
+# features = ["const-fn"]
```

Comment:
Should be just removed completely if it is in stable now.

Reply:
you're right!